src/libstd/fs.rs

```diff
@@ -663,15 +623,17 @@ impl Permissions {
     }
 }
 
-#[unstable(feature = "file_type", reason = "recently added API")]
 impl FileType {
```

Comment:
Sorry, I'm not re-loading before posting... For now, I feel that the enum is small enough that this isn't really an issue however, I can see this becoming an issue in the future as more platforms are added and this enum grows. So I see your point.